builtin/mainmenu/dlg_create_world.lua

```diff
@@ -96,6 +96,8 @@ local function create_world_formspec(dialogdata)
 	if game ~= nil then
 		local gameconfig = Settings(game.path.."/game.conf")
 
+		current_mg = current_mg or gameconfig:get("default_mapgen") or core.settings:get("mg_name")
```

Comment:
From what I can see,  L83 will always evaluate to `nil`.
I'd suggest to remove that line AND:
```suggestion
		current_mg = gameconfig:get("default_mapgen") or core.settings:get("mg_name")
```